project1/src/components/FormikInputs/FormikTextField.tsx

```diff
@@ -12,6 +12,8 @@ interface FormikTextFieldProps {
   label: string;
   margin: PropTypes.Margin;
   type?: string;
+  multiline?: boolean;
```

Comment:
make `FormikTextFieldProps` to extend material-ui textField props interface and pass the rest of the props as it is to `TextField` so we don't need to pass each prop manually